README.md

```diff
@@ -255,7 +255,7 @@ make cluster-down
 
 # Releasing
 
-1. Checkout a public branch
+1. Checkout a public branch (e.g. `master` or `release-*`)
 2. Call `make prepare-patch|minor|major` and prepare release notes
 3. Open a new PR
```

Comment:
Might I suggest change the 2 last bullets into one like so:
- Once the PR is merged, Open a new PR 
- Create a new release in GitHub and attach new manifests and push it to the PR.

Reply:
"Once the PR is merged, Open a new PR" what do you mean by that?

Reply:
there are essentially 2 PRs here (maybe we should call them PR#1 and PR#2 to avoid confusion)
What I meant is, `Once PR#1 is merged, open PR#2` - meaning they shouldn't be opened in parallel*.
then `Create a new release in GitHub and attach new manifests and push it to PR#2.`

*  If this a wrong assumption then we can drop this of course. but at the very least, it needs to be clear what should come into each PR.

Reply:
There is a single PR. The release process looks as following:

- Checkout master
- Call prepare-minor, this creates a new commit with bump of the version and new set of manifests
- Open a PR with this new commit
- If all looks good, merge

Step 4 is not done via a PR. You just open GitHub UI, draft a new Release, update PRs. No review process is present in this step.

Reply:
much clearer to me now. Thanks!